client/src/router/route.ts

```diff
@@ -1,10 +1,12 @@
+/**
+ * @property {string} name Pretty name
+ * @property {string} path Route path
+ * @property {boolean} exact Match this exact route but not children
+ * @property {import('react').Component} component React component to render
+ */
```

Comment:
Was this change to JSDoc intentional?

Reply:
Nope, something must have went wrong with the merge.
85f1ecb